packages/sunburst/src/hooks.ts

```diff
@@ -32,6 +35,8 @@ export const useSunburst = <RawDatum>({
     valueFormat?: DataProps<RawDatum>['valueFormat']
     radius: number
     cornerRadius?: SunburstCommonProps<RawDatum>['cornerRadius']
+    innerRadius?: SunburstCommonProps<RawDatum>['innerRadius']
```

Comment:
The properties should be added to the dependencies of `useMemo`, if you check the live demo on vercel, you can see that changing these values doesn't do anything until you change other properties.